master/internal/resourcemanagers/resource_pool.go

```diff
@@ -422,6 +437,7 @@ func (rp *ResourcePool) receiveJobQueueMsg(ctx *actor.Context) error {
 }
 
 func (rp *ResourcePool) persist() error {
+	// exlude head and tail?
 	fmt.Println("TODO persist rp", rp.config.PoolName)
```

Comment:
can these print statements be removed in favor of a todo comment?

Reply:
I wanted to make it obvious how many times we're getting here as we look through master logs in different cases to see how frequently we're doing this to see if anything would be off. code comment wouldn't expose it in that way but that works too.